ts/features/walletV3/payment/screens/WalletPaymentOutcomeScreen.tsx

```diff
@@ -47,6 +50,18 @@ const WalletPaymentOutcomeScreen = () => {
     outcome
   });
 
+  // This is a workaround to disable swipe back gesture on this screen
+  // .. it should be removed as soon as the migration to react-navigation v6 is completed
+  React.useEffect(() => {
+    // Disable swipe
+    navigation.setOptions({ gestureEnabled: false });
+    navigation.getParent()?.setOptions({ gestureEnabled: false });
+    // Re-enable swipe after going back
+    return () => {
+      navigation.getParent()?.setOptions({ gestureEnabled: true });
+    };
+  }, [navigation]);
+
```

Comment:
Can you add a reference to the ticket to work on later?

Reply:
Absolutely! Added into 86f52584f9264aa3477f75af6a659c3fd7fae345